packages/core/src/messagingProtocols/proteus/ProteusService/ProteusService.ts

```diff
@@ -289,15 +283,14 @@ export class ProteusService {
 
   private async migrateToCoreCrypto(storeEngine: CRUDEngine) {
     const dbName = storeEngine.storeName;
-    const migrationFlag = `${dbName}-corecrypto-ready`;
-    const localStorage = getLocalStorage();
-    if (localStorage.getItem(migrationFlag)) {
+
+    if (cryptoMigrationStore.coreCrypto.isReady(dbName)) {
       return;
     }
 
     this.logger.log(`Migrating data from cryptobox store (${dbName}) to corecrypto.`);
```

Comment:
I'd add a logger to cryptoclient and move these logs there, so we will see them only if we're actually triggering migration, wdyt?